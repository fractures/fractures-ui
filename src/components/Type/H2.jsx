import PropTypes from 'prop-types'
import styled from 'styled-components'

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const H2 = styled.h2`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 1.75rem;
	line-height: 2.1875rem;
`

H2.propTypes = propTypes

export default H2
