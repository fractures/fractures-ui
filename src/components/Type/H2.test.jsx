import H2 from "./H2"

describe("H2", () => {
	const tree = renderer.create(<H2>Extra header</H2>).toJSON()

	it("Matches snapshot", () => {
		expect(tree).toMatchSnapshot()
	})
})
