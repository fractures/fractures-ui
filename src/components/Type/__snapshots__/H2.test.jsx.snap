// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 Matches snapshot 1`] = `
<h2
  className="sc-bdVaJa huXeuD"
>
  Extra header
</h2>
`;
